packages/db/tests/lib/test-execution.js

```diff
@@ -565,7 +565,7 @@ test('getWorkflowNameIntersectFromGranuleIds() returns correct values for single
 
   const results = await getWorkflowNameIntersectFromGranuleIds(knex, [granuleCumulusId]);
 
-  t.deepEqual(results.sort(), ['fakeWorkflow', 'fakeWorkflow2']);
+  t.deepEqual(results.sort(), ['fakeWorkflow', 'fakeWorkflow2'].sort());
```

Comment:
Why do we need to sort the second array with `fakeWorkflow`?

Reply:
As I look at this again, I don't know why, since the second array is already sorted alphabetically it should match `results.sort()`, so I'll remove this

Reply:
5b338537e6e3ebc6ee30bf5c5abdef736ec7a227